README.md

```diff
@@ -0,0 +1,32 @@
+# ftrack OpenAssetIO Plugin
+
+A skeleton OpenAssetIO ManagerPlugin that binds ftrack to the API
+
+> **Warning**
+> This plugin is highly experimental, and subject to significant
+> change.
+
+## Getting started
+
+Place the `plugin` on `$OPENASSETIO_PLUGIN_PATH`. It requires
```

Comment:
[fix] I'd be a bit more explicit here. I had to search through the OpenAssetIO repo to figure out what to do here, eventually finding an example in Simpleresolver. I don't think it's necessarily apparent from "place X on `$OPENASSETIO_PLUGIN_PATH`" that `OPENASSETIO_PLUGIN_PATH` is an environment variable that needs to be set externally.

Having `plugin` in code-quotes here could also be misconstrued, as it implies a direct quote, but there is no directory or file called `plugin`.

Reply:
[This](https://github.com/foundrytom/openassetio-manager-ftrack/pull/1/commits/ce9da004bce1d45d9c213bfebd1916fbd35d6787) better?

Reply:
:+1: